main.ts

```diff
@@ -85,8 +92,15 @@ export default class JupyterPlugin extends Plugin {
 	clients: Map<string, JupyterClient>;
 
 	async postprocessor(src: string, el: HTMLElement, ctx: MarkdownPostProcessorContext) {
+		const extractLanguageFromClass = (str: string) => str.split('-').pop();
```

Comment:
A few questions on this one:

- Given that we're only using this once, can we just use `let language = el.classList[0].split('-').pop()`. 
- Might be worth adding a default of `python` here as well to avoid breaking backwards compatibility.
- Do we know that the `jupyter-[language]` class is always the first one? Or should we search all classes and use the one that has the `jupyter-` prefix?